pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -35,6 +35,10 @@ message ClientConfiguration {
   //
   // Header names must be lower case.
   repeated string forward_metadata = 4;
+
+  // Token-based authentication with Google.
```

Comment:
This should still be updated to no longer mention Google explicitly, right? This can also be used outside of Google.

Reply:
I'm not sure about it, because both methods (`oauth.NewApplicationDefault` and `oauth.NewServiceAccountFromKey`) are Google-specific. Mb in future "google.golang.org/grpc/credentials/oauth" will be extended to support other OAuth providers.

Reply:
Ah, got it. Well, in that case let's just stick to this then.

Reply:
From the other side, https://grpc.io/docs/guides/auth/ already mentions this fact, so I can remove it from comment.

Reply:
added in the next commit